docs/competitive-programming/max-number-from-array.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Maximum number formed from numbers in array
+slug: /competitive-programming/max-number-from-array
+---
+
+## Introduction
+
+To form the maximum number possible from an array of single-digit numbers, we have to sort the array in descending order. The numbers appended in the descending order give the maximum number. However, the same algorithm cannot be used for array containing both two-digit and single-digit numbers. For example, if the array contains two elements, `2` and `21`, then according to the algorithm used for the single digits, if we sort the array in descending order and append them, then the maximum number will be `212`. But this is wrong. The maximum number that can be formed from this array is `221`. Therefore, we need to use a separate algorithm.
```

Comment:
First introduce the problem. What exactly we are trying to do here and what is its relevance. Some of the part can be shifted to the explanation

Reply:
Done. I have written a small introduction to the problem and moved the example to the problem in the Explanation section stating why another algorithm is needed.